addon/components/frost-pick-list.js

```diff
@@ -70,32 +84,58 @@ export default Component.extend({
     return `Showing ${items.length} of ${total} (${amtSelected} selected)`
   },
 
-  // == Functions =============================================================
+  @readOnly
+  @computed('list.@each.' + this.itemKey, 'inputFilter')
```

Comment:
where is this.itemkey coming from? also if you are depending on the list items here, just `list.[]` is enough as a dependent key.